pkg/instance/instance_integration_test.go

```diff
@@ -207,6 +207,46 @@ func TestInstanceHandler(t *testing.T) {
 		time.Sleep(3 * time.Second)
 		k8sClient.AssertPodIsNotRunning(t, deploymentInstance.GroupName, deploymentInstance.Name)
 	})
+
+	t.Run("GetPublicDeployments", func(t *testing.T) {
+		t.Log("Create deployment")
```

Comment:
Can this run in parallel to the other tests? Or we merge this with the other tests we have so that we put the `t.Log("Create public deployment")` block in to `t.Run("Deployment", func(t *testing.T) {` and then the `t.Log("Get public deployments")` at the end of it. As we do with the user integration tests.